setup.py

```diff
@@ -62,14 +62,19 @@
       test_suite='nose.collector',
       packages=find_packages(),
       package_data={
-        'metapool': ['data/*.tsv', 'data/*.xlsx', 'tests/data/*.csv']},
+          'metapool': ['data/*.tsv', 'data/*.xlsx', 'tests/data/*.csv']},
       # addingt all the notebooks fps
       data_files=notebooks_fp,
       install_requires=base,
       extras_require={'test': test,
                       'coverage': coverage,
                       'all': all_deps},
-      entry_points='''
-          [console_scripts]
-          seqpro=metapool.scripts.seqpro:format_preparation_files
-      ''')
+      entry_points={
+          'console_scripts': [
+              'foo=package.module:func',
+              'seqpro=metapool.scripts.seqpro:format_preparation_files',
+              ('seqpro_mf=metapool.scripts.seqpro_mf:format_preparation_'
+               'files_mf'),
+          ],
```

Comment:
Why did this had to change and why use foo?

Reply:
It had to change to add seqpro_mf as an executable that is visible in the path once the package has been installed. the foo line is leftover from the example I used for reference. my bad. :)